CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Master (Unreleased)
 
 - Fix an infinite loop error when `RSpec/ExcessiveDocstringSpacing` finds a description with non-ASCII leading/trailing whitespace. ([@bcgraham])
+- Split `RSpec/FilePath` into `RSpec/SpecFilePathSuffix` and `RSpec/SpecFilePathFormat`. ([@ydah])
```

Comment:
Should we remind people here that while the old `RSpec/FilePath` cop was enabled by default, the two new cops are `pending` and need to be explicitly enabled?